src/ReaderScreen/utils/gutkaScript.js

```diff
@@ -101,13 +114,26 @@ window.addEventListener(
   false
 );
 
-${listener}.onload = () => {
+// Use window.onload for Android reliability
+window.onload = () => {
   if (${theme.mode === "dark"}) {
   //fade event
 fadeInEffect();
 }
 
-  scrollToPosition(); 
+  scrollToPosition();
+  
+  // Minimal scrollbar - works on Android and iOS
```

Comment:
Can you please add some explanation, on the purpose of this code, is this removing an existing div with id `#sb` and then adding another div with same id `sb`?

Reply:
This is added as defensive logic to handle cases where the screen may load multiple times, preventing duplicate scrollbars from being created.